src/Caliburn.Micro.Platform/ActionMessage.cs

```diff
@@ -191,7 +191,7 @@ void ElementUnloaded(object sender, RoutedEventArgs e)
             }
             else currentElement = context.View;
 
-#if NET
+#if NET || NETCOREAPP
```

Comment:
Where is `NETCOREAPP` being defined? Would expect to see something in [`Directory.Build.targets`](https://github.com/Caliburn-Micro/Caliburn.Micro/blob/master/src/Directory.Build.targets).

Reply:
`NETCOREAPP` is a predefined symbol for .NET Core as target framework.
https://docs.microsoft.com/en-us/dotnet/standard/frameworks

Sorry, I did not check the `Directory.Build.targets` file. I have replaced the `NETCOREAPP` symbol with `NETCORE`, which is now defined in [Directory.Build.targets](https://github.com/Caliburn-Micro/Caliburn.Micro/blob/df87adaac6208f07ea8fd727e5c2a2eed75512de/src/Directory.Build.targets). (df87adaac6208f07ea8fd727e5c2a2eed75512de)